src/oops/SOLID/openClosePrinciple/before/taxes/InternTaxCalculator.java

```diff
@@ -0,0 +1,15 @@
+package oops.SOLID.openClosePrinciple.before.taxes;
+
+import oops.SOLID.openClosePrinciple.before.employees.Employee;
+
+public class InternTaxCalculator extends TaxCalculator {
+    private static final int INCOME_TAX_PERCENTAGE = 15;
+    private static final int THRESHOLD = 300000;
+
+    @Override
+    public double calculate(final Employee employee) {
+        return employee.getMonthlyIncome() >= THRESHOLD ?
```

Comment:
Given threshold in the requirements is on the annual income. Not required as such since this exercise is focussed on SRP and OCP, but just a suggestion.

Reply:
He he... yeah 😆  I realised it. but was too lazy to do it after push 🤣